prql-lib/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+extern crate libc;
+
+use libc::{c_char, c_int};
```

Comment:
I think we want this line: https://github.com/prql/prql/blob/main/prql-python/src/lib.rs#L0-L1, to not attempt to build this with wasm

Reply:
I don't know how to prevent that. It is above my Rust knowledge.
Can you tell me what directives/pragmas are needed for that?

Reply:
Found it in python library. I'll add it.

Reply:
Sorry, I just meant — could we put the linked line at the top of this file too?